android/codepush.gradle

```diff
@@ -18,6 +18,12 @@ gradle.projectsEvaluated {
     def buildTypes = android.buildTypes.collect { type -> type.name }
     def productFlavors = android.productFlavors.collect { flavor -> flavor.name }
     if (!productFlavors) productFlavors.add('')
+    def nodeModulesPath;
+    if (project.hasProperty('nodeModulesPath')) {
+        nodeModulesPath = project.nodeModulesPath
```

Comment:
When people override the module path, what location are they expecting to set the path relative to?